_posts/2019-02-15-web-simulator.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: "Web Simulator for Nybble"
+categories: ["Missouri State University"]
+tags: ["CSC596", "JavaScript"]
+author: "Saket Roy"
+---
+
+Working with Dr. Anthony Clark, I am working on building a web simulator for the Nybble, an open-source robotic kitten. The simulator will be written in TypeScript, using the HyperApp Framework and bundled through Parcel. Furthermore, I will be using the three.js 3D library and the cannon.js physics engine to render the simulation. All of the development is front-end with no server-side scripting, and the website will be hosted on github.io
```

Comment:
```suggestion
With Dr. Anthony Clark, I am working on building a web simulator for the Nybble, an open-source robotic kitten. I am writing the simulator in TypeScript, using the HyperApp Framework and Parcel. I will use the Three.js 3D library and the Cannon.js physics engine to render the simulation. All the development is front-end with no server-side scripting, and the website will be hosted on github.io
```

You also need to add links to Nybble, TypeScript, HyperApp, Parcel, Threejs, Cannon.js, and GitHub pages.